updater/library/src/updater.rs

```diff
@@ -1,5 +1,6 @@
 // This file's job is to be the Rust API for the updater.
 
+use std::fmt;
```

Comment:
why is this necessary if we're using `std::fmt` below?

Reply:
In rust you don't ever have to import anything, you can always just fully qualify the name.  So you can just always type std::fmt::Foo if you want, but often you instead "use std::fmt::Foo" to be able to just type "Foo". But similarly you can do "use std::fmt" and then be able to type "fmd::Foo" everywhere.